### PR TITLE
add bomb block and bounce templates

### PR DESCRIPTION
**Pull request by SchwartzGandhi**
Made templates for gaining height in morphball and for destory single bomb blocks
(Can use bombs OR springball)
(can use any bombs or Screw+Knowledge Beginner)
respectively

### CODE REVIEW

File: randovania/games/fusion/logic_database/header.txt [hunk 1]
```diff
@@ -76,6 +76,17 @@ Templates
           Can Use Power Bombs
           Screw Attack and Knowledge (Beginner)
 
+* Can Bounce in Ball:
+      Can Use Bombs or Can Use Springball
+
+* Can Screw Single Bomb Blocks:
+      Trivial
```
Comment (Miepee) [inline randovania/games/fusion/logic_database/header.txt]: is this supposed to be trivbial?

Comment (SchwartzGandhi) [reply randovania/games/fusion/logic_database/header.txt]: oops, ignore that. I spelled the template wrong and made a new one. I already deleted it in the json.

**Comment by Miepee** [comment]
Also IDK if its just me, but i find "can screw single bomb blocks" clearer for the template name than "can break single bomb blocks". could just be me tho.

**Comment by SchwartzGandhi** [comment]
the idea is that you can use bombs OR screw attack, its meant to have a wider range of purpose

**Pull request merged**